src/WebJobs.Script.WebHost/App_Start/WebHostResolver.cs

```diff
@@ -193,6 +180,50 @@ private static ScriptHostConfiguration GetScriptHostConfiguration(string scriptP
             return scriptHostConfig;
         }
 
+        private static void InitializeFileSystem(string scriptPath)
+        {
+            string home = ScriptSettingsManager.Instance.GetSetting(EnvironmentSettingNames.AzureWebsiteHomePath);
+            if (!string.IsNullOrEmpty(home))
```

Comment:
IsAzureEnvironment uses a different app setting (site ID), which is fine, but since we're using the home value, I kept the existing logic. We could make the check using `IsAzureEnvironment` and grab home if true.